src/test/java/org/jabref/logic/importer/FulltextFetchersTest.java

```diff
@@ -69,7 +70,7 @@ public void higherTrustLevelWins() throws MalformedURLException, IOException, Fe
         when(finderHigh.findFullText(entry)).thenReturn(Optional.of(highUrl));
         when(finderLow.findFullText(entry)).thenReturn(Optional.of(lowUrl));
 
-        FulltextFetchers fetcher = new FulltextFetchers(Arrays.asList(finderLow, finderHigh));
+        FulltextFetchers fetcher = new FulltextFetchers(new HashSet<>(Arrays.asList(finderLow, finderHigh)));
```

Comment:
This feels ugly. Can't another constructor be added to FullTextFetchers so that the caller doesn't need to pass a sortedset, but something more easy? Maybe, even `...` could be used so that fetcherLow and fetcherHigh can directly be passed?

Reply:
I think a new constructor is unnecessary. Just remembered that there is a Set.of method..